justfile

```diff
@@ -32,11 +33,18 @@ run-services:
   # if someone has a better solution, be my guest
   sed 's/GATEWAY_IPV4/'` ip addr show eth0 | grep inet | awk '{print $2}' | cut -d/ -f1 | head -n1`'/g' \
     docker/docker-compose.monkey-patch-linux.yml | tee tmp/docker-compose.linux.yml
-  cd docker && docker compose -f docker-compose.yml -f ../tmp/docker-compose.linux.yml up 
```

Comment:
why was this removed?

Reply:
Removed by an accident - will revert it.